examples/general/basic_dead_store/src/lib.rs

```diff
@@ -0,0 +1,147 @@
+#![feature(rustc_private)]
+#![warn(unused_extern_crates)]
+
+extern crate rustc_ast;
+extern crate rustc_hir;
+extern crate rustc_span;
+use clippy_utils::{diagnostics::span_lint_and_help, get_parent_expr};
+
```

Comment:
```suggestion

use clippy_utils::{diagnostics::span_lint_and_help, get_parent_expr};
```
Style nit (combine this `use` with the ones just below it).